GTBitsOfGood/georgiaCORE
# Madeline/edit question design

## Edit Question Design and Leaf Node

Issue Number(s): #41 

What does this PR change and why? Edits leaf node design to be reflected in the preview. Adds home and back button for navigating through preview. Adds progress bar for visualizing where you are in the tree. Switches styling to Chakra UI for ease and later mobile view.

### Checklist


### Critical Changes

None

### Related PRs

None

### How to Test

View trees in preview and navigate with home and back buttons. Home should be the root and back should be previous viewed question. Scroll buttons should shift options over by 1. Error nodes should display error message. URL nodes should display success message and url. Mobile view not implemented yet.
